### DISCUSSION TITLE
How do I search for a particular file across all my repositories?

### DISCUSSION THREAD

**Discussion by blablaalb** [community/community#59556] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I want to find a file by its name within my repositories. I tried this query: 
```
path:/(^|\/)\.gitignore$/ user:blablaalb
```
, but it didn't return anything. 
![image](https://github.com/community/community/assets/55081715/64805059-1ed9-41db-bd46-4ea175e90d60)
Note the _repositories_ section doesn't support `path` qualifier. it just returns all my repositories regardless if they match the query.
![image](https://github.com/community/community/assets/55081715/40d26d51-df44-42e3-903e-48c77afe2591)
Changing `user` to `owner` didn't do anything.
The path qualifier works, because If I remove the `user` qualifier it returns results from across all github repositories. 
How do I narrow down the search to only my repositories? 

---
If my search query is correct I think it might probably means my repositories are not indexed by GitHub.

**Comment by emirhanyagci** [comment]
owner:blablaalb path:/(^|\/)\.gitignore$/

you can try this

**Reply by blablaalb** [reply]
Thanks for the reply. I tried and it didn't work
![image](https://github.com/community/community/assets/55081715/3d5c1cc4-2ae1-47df-8531-398f11840d09)
![image](https://github.com/community/community/assets/55081715/8aa548fd-f3c5-4c6e-b47b-706afe13b13f)

**Comment by emirhanyagci** [comment]
path:readme.txt user:emirhanyagci

can you try this with your name

**Reply by emirhanyagci** [reply]
and dont forget you shuld check repos section dont stay in the code section (can you submit answer if i fix)

**Reply by blablaalb** [reply]
The repos section doesn't support `path` qualifier, it returns all repositories regardless if they match the query.  The `path:readme.md user:blablaalb` query returned only one result, even though I have several repositories with the `readme.md` file.  It probably means my repositories are not  indexed by GitHub.

**Reply by GhostOf0days** [reply, marked as answer]
You are probably right in that it is your repositories not being indexed by Github. Typing that for me returns all my ReadMe's while it only returns one for you.
